src/rpc/core_rpc_server.cpp

```diff
@@ -1075,12 +1039,7 @@ namespace cryptonote
   //------------------------------------------------------------------------------------------------------------------------------
   bool core_rpc_server::on_get_last_block_header(const COMMAND_RPC_GET_LAST_BLOCK_HEADER::request& req, COMMAND_RPC_GET_LAST_BLOCK_HEADER::response& res, epee::json_rpc::error& error_resp)
   {
-    if(!check_core_busy())
-    {
-      error_resp.code = CORE_RPC_ERROR_CODE_CORE_BUSY;
-      error_resp.message = "Core is busy.";
-      return false;
-    }
+    CHECK_CORE_READY();
```

Comment:
Why is this check added here, but not for the other functions?

Reply:
I deemed the "last" wording to be "not whatever's currently being synced", but "last known". So making no sense while syncing.
That's a bit subjective, and I'd be fine removing it (or making it into another patch).